### PR TITLE
creating view docs

### PR DESCRIPTION
**Pull request by ktabors**
Closes https://jira.corp.adobe.com/browse/RSP-1740

## 🧢 Your Team:
RSP

### CODE REVIEW

File: packages/@react-spectrum/image/docs/Image.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+### Internationalization
+
+To internationalize an Image, a localized string should be passed to the `alt` prop.
+
+## Props
+
+<PropTable component={docs.exports.Image} links={docs.links} />
+
+## Visual Options
```
Comment (ktabors) [inline packages/@react-spectrum/image/docs/Image.mdx:58]: The `loaded` and `isPlaceholder` props don't have styles in the code yet so I left them out of the docs.

File: packages/@react-types/view/src/content.d.ts [hunk 2]
```diff
@@ -14,5 +14,8 @@ import {DOMProps, StyleProps} from '@react-types/shared';
 import {ReactNode} from 'react';
 
 export interface ContentProps extends DOMProps, StyleProps {
+  /**
+   * Content content
```
Comment (ktabors) [inline packages/@react-types/view/src/content.d.ts:18]: This feels like a joke, but is following the other docs.

Comment (snowystinger) [reply packages/@react-types/view/src/content.d.ts:18]: easter egg it, so meta

File: packages/@react-spectrum/view/docs/Content.mdx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Content
+
+The Content component is used as a child to other
+components providing formatting and semantic meaning as explained in
+[Dialog](Dialog.html#content). It is
+important to note that the Content element
+accepts any renderable node, not just strings.
+
+A Content is used to specify the content of a component. Examples of usage
```
Comment (LFDanLu) [inline packages/@react-spectrum/view/docs/Content.mdx:47]: Perhaps "Example usage can be found in [Dialog](Dialog.html#example) and [IllustratedMessage](IllustratedMessage.html#example)."

Comment (LFDanLu) [inline packages/@react-spectrum/view/docs/Content.mdx:47]: Perhaps we could try combining the first and third sentence here just like in Image.

File: packages/@react-spectrum/view/src/Footer.tsx [hunk 4]
```diff
@@ -31,5 +31,8 @@ function Footer(props: FooterProps, ref: DOMRef) {
   );
 }
 
+/**
+ * Footer is used to position footer content with a component.
```
Comment (LFDanLu) [inline packages/@react-spectrum/view/src/Footer.tsx:35]: "with" -> "within"

File: packages/@react-spectrum/image/docs/Image.mdx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+```tsx example
+<Image src="https://i.imgur.com/Z7AzH2c.png" alt="Sky and roof" />
+```
+
+## Content
+
+The Image component is used as a child to other components providing formatting
+and semantic meaning.
+
+An Image is used to add images to components. Example of usage [Dialog](Dialog.html#examples).
```
[the remark this answers is not in the document]
Comment (LFDanLu) [reply packages/@react-spectrum/image/docs/Image.mdx:44]: Same comment for the other components as well

File: packages/@react-types/view/src/content.d.ts (hunk 2 above)
Comment (solimant) [reply packages/@react-types/view/src/content.d.ts:18]: content

File: packages/@react-types/image/src/index.d.ts [hunk 6]
```diff
@@ -13,11 +13,29 @@
 import {DOMProps, StyleProps} from '@react-types/shared';
 
 export interface ImageProps {
-  loaded?: boolean, // not real, just makes ts happy for now
-  isPlaceholder?: boolean, // same thing
+  /**
+   * Whether the Image should be displayed with a loaded style.
+   */
+  loaded?: boolean,
```
Comment (solimant) [inline packages/@react-types/image/src/index.d.ts:19]: I know this is out of this PR's scope, but shouldn't this be `isLoaded` following the `isSomething` convention for booleans?

**Pull request merged**